educator/src/Dscp/Educator/Web/Student/Handlers.hs

```diff
@@ -24,17 +23,14 @@ type StudentApiWorkMode m =
     ( MonadStudentAPIQuery m
     )
 
--- TODO [DSCP-141]: remove these two
-oneGeek :: Student
-oneGeek = studentEx
-
+-- TODO [DSCP-141]: remove this
 oneGeekSK :: SecretKey
 oneGeekSK = studentSKEx
 
 studentApiHandlers
     :: forall m. StudentApiWorkMode m
-    => StudentApiHandlers m
-studentApiHandlers =
+    => Student -> StudentApiHandlers m
+studentApiHandlers oneGeek = traceShow oneGeek $
```

Comment:
Please, don't call students "geeks", some can become offended after they read the code. This is open source, after all.

Reply:
Loool, it just was temporal constant so I called it funny, now you'd better to replace it with something plain ))